### DISCUSSION TITLE
dropping Python 3.7 support

### DISCUSSION THREAD

**Discussion by pubpub-zz** [py-pdf/pypdf#2735] [General]
Python 3.7 has reached EoL in June 23.
We are 1 year later, It could be a good time to stop supporting 3.7

pros:
* will allow to move to pillow 10.4

cons:
* (None)

Opinions are welcomed 😀

**Comment by stefan6419846** [comment]
Isn't this a duplicate of #2418?

If it is just about *Pillow*, I do not see the issue - we could just disallow the (at least in some cases) known to be broken version 10.3.0. The larger benefit of dropping Python 3.7 would be to use more recent package versions in CI more easily as discussed in #2455 and #2574.

**Reply by pubpub-zz** [reply]
agree. Closing this thread as duplicate. can you republish in #2418?
